stats/Record.md

```diff
@@ -51,7 +51,10 @@ contextual information of an exemplar, for example trace id, span id or dropped
 ## Recording Stats
 
 Users should record Measurements against a context, either an explicit context or the implicit 
-current context. Tags from the context are recorded with the Measurements if they are any.
+current context. Tags from the context are recorded with the Measurements if they are any. 
+When recording against an explicit context, implmentations should allow users to add extra tags,
+and those tags should be non-propagating.
```

Comment:
We haven't really defined what non-propagating means. I think it would be enough to say here that the extra tags are not added to the context.

Reply:
Sounds good, updated.